source/client-side-encryption/client-side-encryption.rst

```diff
@@ -1339,7 +1339,7 @@ NOTE: The Range algorithm is experimental only. It is not intended for public us
 contentionFactor
 ^^^^^^^^^^^^^^^^
 contentionFactor only applies when algorithm is "Indexed" or "RangePreview".
-It is an error to set contentionFactor when algorithm is not "Indexed".
+It is an error to set contentionFactor when algorithm is not "Indexed" or "RangePreview".
```

Comment:
Other options were amended to say "or RangePreview", so I think it was just an oversight that `contentionFactor` was not. The prose test for range index certainly uses this option.